code/modules/clothing/head/helmet.dm

```diff
@@ -13,6 +13,7 @@
 	resistance_flags = NONE
 	flags_cover = HEADCOVERSEYES
 	flags_inv = HIDEHAIR
+	snug_fit = TRUE

```

Comment:
Is making a real security helmet a hat easily removable a bad idea?

Reply:
snug_fit means it can't be knocked off

Reply:
Oh shit I'm so dumb sorry